go.mod

```diff
@@ -1,19 +1,21 @@
 module github.com/rook/kubectl-rook-ceph
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.1
```

Comment:
Why was the toochain needed as a separate version from go above? It's nice if we can just declare `1.22` and it automatically picks up the latest patch.

Reply:
Let's discuss this in huddle,
 the k8s  0.30 api are bringing those changes either we have to mention `toolchain go.*` or we have to use explicit version of go in go.mod like `go 1.22.1`